pages/src/common/colors.less

```
@@ -17,14 +18,15 @@
 @body-bg: #f8f8f8;
 @body-fg: rgba(22, 0, 27, 0.9);
 
+
 @link-fg: @primary-blue;
 @link-hover-fg: @secondary-blue;
 @link-hover-underline: @secondary-blue;
 
-@landing-bg-color: #131a2e;
+@landing-bg-color: @primary-bg;
 @landing-fg-color: #fff;
 
-@nav-dark-bg: @landing-bg-color; //#232429;
+@nav-dark-bg: @primary-bg; //#232429;
```

cosmetic: do we need this comment? 
```suggestion
@nav-dark-bg: @primary-bg;
```